### ISSUE TITLE
Button presses device '00:bb:3a:04:de:59' not detected.

### ISSUE THREAD

**Issue by mwittig** [mwittig/pimatic-amazing-dash-button#4]
This popped up here: https://forum.pimatic.org/topic/2427/amazing-dash-button-one-of-two-buttons-not-recognized

If the the device is created manually with given MAC, button presses will not be detected for that device as the MAC is not added to the filter. This is a bug.

**Comment by mwittig** [comment]
Checked 'MAC Address Block Large (MA-L)' at <https://regauth.standards.ieee.org/standards-ra-web/pub/view.html#registries>. The vendor is is marked as being private as companies may deny public disclosure. Strange, why Amazon is starting with such bad habbits.

```
00-BB-3A   (hex)		        Private
00BB3A     (base 16)		Private
```

**Comment by mwittig** [comment]
Released pimatic-amazing-dash-button@0.9.10 which adds support for MAC `00:bb:3a:04:de:59`, however, the actual issue is remaining and should be fixed.

**Comment by mwittig** [comment]
Released pimatic-amazing-dash-button@0.9.14
